ppx/jsx-ppx/tyxml_jsx.ml

```diff
@@ -16,6 +17,58 @@ let make_html_attr_name name =
     | "begin_" -> "begin"
     | "end_" -> "end"
     | "in_" -> "in"
+    (* accessibility *)
+    | "ariaDetails" -> "aria-details"
+    | "ariaDisabled" -> "aria-disabled"
+    | "ariaHidden" -> "aria-hidden"
+    | "ariaInvalid" -> "aria-invalid"
+    | "ariaKeyshortcuts" -> "aria-keyshortcuts"
+    | "ariaLabel" -> "aria-label"
+    | "ariaRoledescription" -> "aria-roledescription"
+    (* Widget Attributes *)
+    | "ariaAutocomplete" -> "aria-autocomplete"
+    | "ariaChecked: true|false|mixed, /* https" -> "aria-checked"
```

Comment:
hmm I'm not sure this one will work

Reply:
I was a bit too fast with my find and replace...